src/components/CreateAgencyBtn/CreateAgencyBtn.css

```diff
@@ -0,0 +1,24 @@
+button {
+  border: none;
+  background: none;
+  display: flex;
+  justify-content: space-between;
+}
+.res-circle {
```

Comment:
Can you add whitespaces between each of these style elements? And also potentially inline comments for each or a file header to just clarify what the styling is for